### ISSUE TITLE
Remote content exceptions not processed correctly in TB42

### ISSUE THREAD

**Issue by ckempo** [protz/thunderbird-conversations#1045]
Using TB 42 beta with Conversations 2.10.2; when a message with remote content is displayed, the warning message "Remote content was hidden: show remote content - always show remote content from me@you.com" is displayed, but if I click the "always show icon", no entry is added to the permitted list. I suspect this is a change in TB as there used to be a "display remote content from this sender" checkbox in the address book, which is no longer there, and the Tools>Options>Privacy>Mail Content>Exceptions list seems to replace it. If I manually add sites and addresses here, remote content is displayed as expected.

To reproduce:

1.Display email with remote content, where sender is not in the exception list
Click "always show remote content" in the header bar

**Comment by protz** [comment]
Correct. I think they changed something on the Thunderbird side in reaction to a change on the Firefox side, so now I need to sync up with the changes on the Thunderbird side

**Comment by tdsnowden** [comment]
Any idea on the timeframe of this bug? Same problem here - not clamoring, just curious.

**Comment by protz** [comment]
I'm short on time right now, so any help would be appreciated...

**Comment by tdsnowden** [comment]
Any pointers? I don't mind helping but haven't dug into mozilla plugins before.

**Comment by protz** [comment]
Ok this has to be resolved urgently, as the next version of Thunderbird comes out next week, and I need to have a working build on addons.mozilla.org by then.

This is the bug that broke us: https://bugzilla.mozilla.org/show_bug.cgi?id=1193200

This is how they fixed it: https://bugzilla.mozilla.org/attachment.cgi?id=8666387&action=diff

This is where we handle it: https://github.com/protz/thunderbird-conversations/blob/master/modules/message.js#L909

@Dexterp37 any chance you could into it? @tdsnowden perhaps? I'm *really* short on time this week...

Thanks,

Jonathan

**Comment by Dexterp37** [comment]
That was quick , I was just about to look at it :-)

**Comment by tdsnowden** [comment]
Just saw it ... that was fast!
